Use of undeclared type 'CGFloat'

I tried building a static library, but I'm getting lots of compiler errors from what look to be missing UIKit imports:

```
/ios/Pods/Pulley/PulleyLib/PulleyViewController+Nested.swift:11:67: error: use of undeclared type 'CGFloat'; did you mean to use 'CGFloat'?
    public func collapsedDrawerHeight(bottomSafeArea: CGFloat) -> CGFloat {
                                                                  ^~~~~~~
                                                                  CGFloat
```

--- a/PulleyLib/PulleyViewController+Nested.swift
+++ b/PulleyLib/PulleyViewController+Nested.swift
@@ -5,7 +5,7 @@
 //  Created by Ethan Gill on 8/1/17.
 //
 
-import Foundation
+import UIKit
 
 extension PulleyViewController: PulleyDrawerViewControllerDelegate {
     public func collapsedDrawerHeight(bottomSafeArea: CGFloat) -> CGFloat {

